### DISCUSSION TITLE
vmware tools not installing

### DISCUSSION THREAD

**Discussion by gbrodt** [Security-Onion-Solutions/securityonion#2264] [Unsupported Versions] [answered]
On SO 2-3-10  the vmware tools are not installing. I've downloaded and extracted the tar file to a /tmp location. When I run ./vmware-isntall.pl I am getting a permission denied.  I have 777.

**Comment by gbrodt** [comment]
I did get the open-vm-tools to install through yum install open-vm-tools

**Comment by dougburks** [comment, marked as answer]
VMware supports and recommends open-vm-tools. 

From https://kb.vmware.com/s/article/2073803:
```
VMware recommends using open-vm-tools redistributed by operating system vendors.
```

**Comment by gbrodt** [comment]
Thank you for the response and reference – and all your great work!

VMware supports and recommends open-vm-tools.

From https://kb.vmware.com/s/article/2073803:

VMware recommends using open-vm-tools redistributed by operating system vendors.
